### PR TITLE
CAN-1185

### PR DESCRIPTION
**Pull request by vikasranatalentelgia**
Manage Support: clicking on camp name should take user to topic detail page

### CODE REVIEW

File: src/components/ComponentPages/ManageSupport/index.tsx [hunk 1]
```diff
@@ -53,6 +53,8 @@ const ManageSupport = () => {
     const body = { topic_num: topicNum };
 
     let res = await getAllUsedNickNames(topicNum && body);
+    console.log(res, "pop");
```
Comment (akuks) [inline src/components/ComponentPages/ManageSupport/index.tsx]: please remove console.log

File: src/components/ComponentPages/UserProfile/_test_/userProfile.test.tsx [hunk 2]
```diff
@@ -33,20 +33,30 @@ const noData = false;
...
-    expect(screen.getByText(labels.city)).toBeTruthy();
-    expect(screen.getByText(labels.country)).toBeTruthy();
-    expect(screen.getByText(labels.zipcode)).toBeTruthy();
+    render(
+      <UserProfileDetails
+        profileData={profileData}
+        userSupportedCampsList={userSupportedCampsList}
+      />
+    );
+    expect(screen.queryByTestId(labels.name)).toBeNull();
```
Comment (akuks) [inline src/components/ComponentPages/UserProfile/_test_/userProfile.test.tsx:54]: Can we create an array and then iterate?

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/ManageSupport/index.tsx (hunk 1 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ManageSupport/index.tsx]: Done

File: src/components/ComponentPages/UserProfile/_test_/userProfile.test.tsx (hunk 2 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/UserProfile/_test_/userProfile.test.tsx:54]: Need to check if it can be done in test cases.

**Review by akuks — APPROVED** [review]

**Pull request merged**